q3tiledaligner/test/au/edu/qimr/tiledaligner/util/TARecordUtilTest.java

```diff
@@ -2659,6 +2659,15 @@ public void longsOverlapping2() {
 		assertEquals(false, TARecordUtil.doGenomicPositionsOverlap(1994, 2121, 2011, 2129, 149));
 	}
 	
+	@Test
```

Comment:
May I suggest a test each for the < and > case as well?

Reply:
thanks for the suggestion - done